cookbook/integration_langchain.ipynb

```diff
@@ -378,15 +378,19 @@
     },
```

Comment:
<p>lets focus on this example.</p><p> </p><p>add single line comments to update_current_trace and get_current_langchain_handler </p><p> </p><p>remove params from main function, just execute it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>